enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/util/FilterUtils.java

```diff
@@ -516,26 +513,6 @@ public static String buildUsernameWithTenant(String username, String tenantDomai
         return username;
     }
 
-    /**
-     * Get the API related throttling tier for auth context.
-     * If API level present, it will be returned. If not resource level
-     * would be returned. If both not present, UNLIMITED tier would be returned.
-     * @param requestContext Request context
-     * @return string format API tier.
-     */
-    public static String getAPILevelTier(RequestContext requestContext) {
-        String apiTier = requestContext.getMatchedAPI().getTier();
-        String resourceTier = requestContext.getMatchedResourcePath().getTier();
-
-        if (!apiTier.isEmpty() && !ThrottleConstants.UNLIMITED_TIER.equalsIgnoreCase(apiTier)) {
-            return apiTier;
-        }
-        if (!resourceTier.isBlank()) {
-            return resourceTier;
-        }
-        return ThrottleConstants.UNLIMITED_TIER;
-    }
-
```

Comment:
This method becomes unused with this PR. The scenario is already handled in ThrottleFilter.